ingest/bin/post_process_metadata.py

```diff
@@ -0,0 +1,91 @@
+#! /usr/bin/env python3
+
+"""Reformat pandas DataTables for a pathogen build.
+
+Expecting one argument, a NCBI Virus metadata.tsv file
+"""
+# ===== Dependencies
+import argparse
+import os
+import sys
+
+import numpy as np
+import pandas as pd
+
+
+def parse_args():
+  # Main help command
+  parser = argparse.ArgumentParser(
+    description = "USAGE: python post_process_metadata.py --metadata 'data/metadata.tsv'"
+  )
+  # Add first argument
+  parser.add_argument(
+    "--metadata",
+    help = "NCBI Virus metadata.tsv file.",
+    required = True
+  )
+  parser.add_argument(
+    "--outfile", default="processed_metadata.tsv",
+    help = "Processed metdata file name. [default: processed_metadata.tsv] ",
+    required = False
```

Comment:
You might want to make this argument required and remove the default, to avoid unpleasant surprises for the user who accidentally forgets to provide the argument and ends up overwriting their output or not knowing where the output went. Since we usually explicitly define inputs and outputs from the calling context (e.g., Snakemake) for better readability and workflow functionality, it doesn't cost much user effort to make this required.

Reply:
Okay, argued myself to your view point. Will make `--outfile` required

* https://softwareengineering.stackexchange.com/questions/63908/default-values-are-they-good-or-evil

Reply:
And since I've already been burned by defaults here: https://github.com/nextstrain/dengue/pull/6/commits/3dac7feeabd496a8561a4f5c716f99a871dc8b5d